persistence/src/main/scala/org/apache/pekko/persistence/snapshot/japi/SnapshotStore.scala

```diff
@@ -33,12 +34,12 @@ abstract class SnapshotStore extends SSnapshotStore with SnapshotStorePlugin {
     doLoadAsync(persistenceId, criteria).asScala.map(option)
 
   override final def saveAsync(metadata: SnapshotMetadata, snapshot: Any): Future[Unit] =
-    doSaveAsync(metadata, snapshot).asScala.map(_ => ())
+    doSaveAsync(metadata, snapshot).asScala.map(ConstantFun.unit)
```

Comment:
map(ConstantFun.unit)(org.apache.pekko.dispatch.ExecutionContexts.parasitic)

Reply:
`import context.dispatcher` - shouldn't this be enough?

Or is there a good reason to prefer `parasitic`?

Reply:
Yes, `parasitic` is the same thread, with which we can avoid a context switching.